pkg/virt-controller/watch/snapshot/snapshot_test.go

```diff
@@ -674,6 +710,37 @@ var _ = Describe("Snapshot controlleer", func() {
 				controller.processVMSnapshotWorkItem()
 			})
 
+			It("should update VirtualMachineSnapshotStatus Indications if vm is running", func() {
```

Comment:
I can't find where indications are added to the snapshot

Reply:
it should be added in the code itself and then be compared to the "updatedSnapshot" created in line 721, which we update its expected indications in lines 727-728